### PR TITLE
Mizu tap - adding logs for troubleshooting

### PR DESCRIPTION
**Pull request by IgorGov**

### CODE REVIEW

File: cli/cmd/tapRunner.go [hunk 1]
```diff
@@ -337,7 +340,7 @@ func syncApiStatus(ctx context.Context, cancel context.CancelFunc, tappingOption
 		default:
 			err = controlSocket.SendNewTappedPodsListMessage(currentlyTappedPods)
 			if err != nil {
-				fmt.Printf("error Sending message via control socket %s\n", err)
+				fmt.Printf("error Sending message via control socket %v, error: %s\n", controlSocketStr, err)
```
Comment (gadotroee) [inline cli/cmd/tapRunner.go:343]: maybe this should be debug

Comment (gadotroee) [reply cli/cmd/tapRunner.go:343]: Just because it is CLI and it will be visible to the user (and I'm not sure we want to write the websocket url

**Review by gadotroee — APPROVED** [review]

**Review by gadotroee — CHANGES_REQUESTED** [review]
Are you sure you commit everything? 
I think the we missing changes in go.mod / go.sum in cli

File: cli/cmd/tapRunner.go (hunk 1 above)
Comment (IgorGov) [reply cli/cmd/tapRunner.go]: changed it to debug

**Review by gadotroee — APPROVED** [review]

**Pull request merged**